Library/Homebrew/cmd/install-vendor.sh

```diff
@@ -0,0 +1,123 @@
+# Hide shellcheck complaint:
+# shellcheck source=/dev/null
+source "$HOMEBREW_LIBRARY/Homebrew/utils/lock.sh"
+
+if [[ -n "$HOMEBREW_OSX" ]]
+then
+  if [[ "$HOMEBREW_PROCESSOR" = "Intel" ]]
+  then
+    ruby_URL="https://github.com/xu-cheng/homebrew-portable/releases/download/20160512/portable-ruby-2.0.0-p648.i386-darwin9.tar.gz"
```

Comment:
As mentioned below, Bintray feels like it may be a better fit for storage here and this repo should be moved into the Homebrew org before we merge this.